Components/landingPage.css

```diff
@@ -0,0 +1,63 @@
+.main-slider-container{
+    background-image:url('/Assests/Book-Landing-Image.png');
+    width:100%;
+    height:25rem;
+    background-repeat: no-repeat;
+    margin-top: 2rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: var(--color-gray-100);
+}
+.slider-icon{
+    font-size: 4rem;
+    cursor: pointer;
+    color: var(--color-gray-600)
+}
+.slider-icon:hover{
+    background-color: var(--color-gray-200);
+}
+.slider-content{
+    margin-right: 40rem;
```

Comment:
don't use large margin values it will affect later when you integrate more components into your project

Reply:
made all requested changes